src/models/events.ts

```diff
@@ -333,6 +333,27 @@ export type DestroyEventQueryParams = CreateEventQueryParams;
  */
 export type SendRsvpQueryParams = FindEventQueryParams;
 
+/**
+ * Interface representing the query parameters for importing events.
+ */
+export interface ListImportEventQueryParams extends ListQueryParams  {
+  /**
+   * (Required) Calendar ID to import events from. "primary" is a supported value indicating the user's primary calendar.
+   * Note: "primary" is not supported for iCloud.
+   */
+  calendarId: string;
+  /**
+   * Filter for events that start at or after the specified time, in Unix timestamp format.
+   * Defaults to the time of the request.
+   */
+  start?: number;
+  /**
+   * Filter for events that end at or before the specified time, in Unix timestamp format.
+   * Defaults to one month from the time of the request.
+   */
+  end?: number;
+}
+
```

Comment:
I think we are missing the `select` parameter here

Reply:
It should be inherited from `ListQueryParams`